frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -279,24 +270,53 @@ const TestCardForm = ({
     return null;
   };
 
-  const getAOEError = () => {
+  const whichAOEFormOption = useAOEFormOption(state.deviceId, devicesMap);
+  const whichAoeFormRequiredForPositives =
```

Comment:
What if we modified `useAOEFormOption` to also return whether the form is required for positives? Since we'll already have the logic in `useAOEFormOption` to determine which form to return I think it'll be easier to track that info right next to whether or not that form is required for positives.

```
const { whichAOEFormOption, aoeRequiredForPositives }= useAOEFormOption(state.deviceId, devicesMap);
```

Reply:
taking a look at this again I think the best way to do this is to separate the display logic away from the selection / form option. I'm going to elect to type the hook to always return a value of the `AOEFormOption` enum and then move the display logic to a separate function. That way we can keep those two things decoupled (and just use the one variable for the switch logic for what form to use!)

Lmk if you have other suggestions / thoughts on this though